tests/integration_tests/security/api_tests.py

```diff
@@ -137,6 +137,149 @@ def test_post_guest_token_bad_resources(self):
 
         self.assert400(response)
 
+    @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
+    @with_config({"GUEST_TOKEN_VALIDATOR_HOOK": None})
+    def test_guest_token_validator_hook_unconfigured(self):
+        """
+        Security API: Test the API works normally with GUEST_TOKEN_VALIDATOR_HOOK
+        configured to None.
+        """
+        self.dash = db.session.query(Dashboard).filter_by(slug="births").first()
+        self.embedded = EmbeddedDashboardDAO.upsert(self.dash, [])
+        self.login(ADMIN_USERNAME)
+        user = {"username": "bob", "first_name": "Bob", "last_name": "Also Bob"}
```

Comment:
This could do with being a bit more DRY, looks like you're using the same user, resource and rls rule in almost all of these tests so I'd suggest just defining them once, or using pytest fixtures.

Basic user details and resource fixtures seem like things we must already have test fixtures for so you can probably reuse something existing.

Reply:
Did some test consolidation there.